model/lib/schema.mustache

```diff
@@ -17,4 +17,7 @@ exports.filter = (name, params) => {
   }, {});
 };
 
-exports.validate = jsen(schema);
+exports.getValidator = (name) => {
```

Comment:
Once this library is a bit more set in stone, API changes like this will need to be approached more carefully, fine for now though mostly.